Geometry/CSCGeometryBuilder/src/CSCGeometryParsFromDD.cc

```diff
@@ -17,6 +28,17 @@
 
 #include <FWCore/MessageLogger/interface/MessageLogger.h>
 
+#include <DetectorDescription/DDCMS/interface/DDFilteredView.h>
```

Comment:
same here for all includes

Reply:
Done

Reply:
@slomeo Have you pushed all your changes?

Reply:
I missed something. Now it is fixed